test/tsGeneralTest.js

```diff
@@ -292,6 +302,16 @@ describe('Timeseries - General', function () {
                 {
                     return;
                 }
+                // FIXME(Sidney): Same for symbols
+                if ((current.name == 'symbol' && next.name == 'string') || (current.name == 'string' && next.name == 'symbol'))
+                {
+                    return;
+                }
+                // FIXME(Sidney): Same for symbols
+                if ((current.name == 'symbol' && next.name == 'blob') || (current.name == 'blob' && next.name == 'symbol'))
+                {
+                    return;
+                }
```

Comment:
Hum? Make the 3 `if`s a single `if` as they check the same thing... And... oh boy.
Have you checked all the combinations? :)

Reply:
I cleaned that a bit ^^